README.md

```diff
@@ -21,13 +21,34 @@ If you are already part of the DACCS Network and details about your node have ch
 In which case, please submit the pull request with the new URL and the Executive Committee will approve it quickly.
 
 When making changes to the node_registry.json file please only change or add the following values:
-  - url (the publicly accessible URL of your node)
-  - date_added (the date the node was originally added to the network, this should only be updated once)
-  - affiliation (the name of your organization, optional)
-  - description (a short description of your node, optional)
-  - icon_url (a publicly accessible url to an icon image for your node, optional)
-  - location (latitude and longitude of your organization, optional)
-  - contact (an email address that can be used by users to contact you if they have questions about your node)
+  - `date_added` (the date the node was originally added to the network, this should only be updated once)
+  - `affiliation` (the name of your organization, optional)
+  - `description` (a short description of your node, optional)
+  - `location` (latitude and longitude of your organization, optional)
+  - `contact` (an email address that can be used by users to contact you if they have questions about your node)
+  - `links -> version` (see links table below)
+  - `links -> collection` (see links table below)
+  - `links -> service` (see links table below)
```

Comment:
JSON path notation would be better (e.g.: `links[*].version`)

Reply:
Wouldn't the syntax be: `links[?(@.rel == "service")]`

Reply:
You're right. 
Maybe `links[*].rel == "service"` is sufficient as well? More readable at least.

Reply:
I agree that it's more readable but it still might be confusing to use a non-standard syntax (I agree that the `links -> service` syntax was bad too).

I'm going to rewrite this a bit so that we just use plain language here and avoid the issue of using this sort of notation.

Reply:
I like the rewording. It is the most concise and easier to understand by anyone. 👍